packages/instrumenter/src/mutators/string-literal-mutator.ts

```diff
@@ -8,17 +8,17 @@ export const stringLiteralMutator: NodeMutator = {
   name: 'StringLiteral',
 
   *mutate(path) {
-    if (path.isTemplateLiteral()) {
+    if (path.isTemplateLiteral() && isValidParentForTemplateLiteral(path)) {
       const replacement = path.node.quasis.length === 1 && path.node.quasis[0].value.raw.length === 0 ? 'Stryker was here!' : '';
       yield types.templateLiteral([types.templateElement({ raw: replacement })], []);
     }
-    if (path.isStringLiteral() && isValidParent(path)) {
+    if (path.isStringLiteral() && isValidParentForStringLiteral(path)) {
       yield types.stringLiteral(path.node.value.length === 0 ? 'Stryker was here!' : '');
     }
   },
 };
 
-function isValidParent(child: NodePath<babel.types.StringLiteral>): boolean {
+function isValidParentForStringLiteral(child: NodePath<babel.types.StringLiteral>): boolean {
   const parent = child.parent;
   return !(
```

Comment:
This method now contains a lot of duplicate code. Can we call `isValidParentForTemplateLiteral` from this method instead? (so share the shared implementation). Or let them both call a `isValidParent` with the shared stuff in it?